leaderboard/views.py

```diff
@@ -14,7 +14,7 @@
 
 def calculate_rankings(company_dict):
     ranks = {}
-    ranks['percent_green_commuters'], ranks['percent_participation'], ranks['percent_green_switches'], ranks['percent_healthy_switches'], ranks['percent_avg_participation'] = [],[],[],[],[]
+    ranks['percent_green_commuters'], ranks['percent_participation'], ranks['percent_green_switches'], ranks['percent_healthy_switches'], ranks['percent_avg_participation'] = [], [], [], [], []
```

Comment:
author's note: I know this wouldn't have been tested; I was attempting to kick a build. Then realized I needed to do it on my fork.